extension/src/experiments/index.ts

```diff
@@ -348,11 +348,13 @@ export class Experiments extends BaseRepository<TableData> {
   }
 
   public async selectExperiments() {
-    const experiments = this.experiments.getExperimentsWithCheckpoints()
+    const experiments = [
+      ...this.experiments.getWorkspaceAndCommits(),
+      ...this.experiments.getExperiments()
+    ]
```

Comment:
[F] Until I know exactly what I need in the model I am going to try and avoid adding extra functions. It seems like there is an awful lot of redundancy in there.